lib/active_merchant/billing/affirm.rb

```diff
@@ -0,0 +1,49 @@
+require 'affirm'
+
+module ActiveMerchant #:nodoc:
```

Comment:
But why do we use both Spree::Gateway and ActiveMerchant Gateway? Active merchant provides gateways, but the interface solidus uses is Spree::Gateway. How it's implemented past Spree::Gateway doesn't concern Solidus. In some solidus gateways we use active merchant gateways just to avoid reinventing the wheel by using gateways provided by AM. 

If we dont already have what we want, we just create a gateway like the one I linked.